doc/source/user_guide/io.rst

```diff
@@ -48,6 +48,71 @@ The pandas I/O API is a set of top level ``reader`` functions accessed like
 CSV & text files
 ----------------
 
+.. _io-colab:
+
+Loading data in Google Colab
+----------------------------
+
+Google Colab is a commonly used runtime for pandas users. Below are concise, copy-pasteable ways to load data into pandas while working in Colab.
+
+Uploading files from your local machine
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Use the Colab file upload widget and then read the uploaded file with :func:`pandas.read_csv`:
+
+.. code-block:: python
+
+    from google.colab import files
+    import pandas as pd
+
+    uploaded = files.upload()  # pick file(s) from your machine
+    df = pd.read_csv('your_file.csv')
+
+
+Mounting Google Drive
+^^^^^^^^^^^^^^^^^^^^^^
+
+To access files you stored in Google Drive:
+
+.. code-block:: python
+
+    from google.colab import drive
+    import pandas as pd
+
+    drive.mount('/content/drive')
+    df = pd.read_csv('/content/drive/MyDrive/path/to/your_file.csv')
```

Comment:
Can we explain how to get the path?

https://python-public-policy.afeld.me/en/columbia/assignments/open_ended.html#reading-files-in-python